content/markdown/concepts/blockquotes/blockquotes.md

```diff
@@ -67,12 +67,10 @@ Blockquotes can contain other Markdown formatted elements. Not all elements can
 
 The output would be:
 
-```markdown
 > ### The quarterly results look great
 >
 > - Dogecoin was off the charts!
 > - Apes together stonk!
 > - $AMC to the moon!
 >
 >   _Everything_ is going according to **the plan**.
```

Comment:
```suggestion
> _Everything_ is going according to **the plan**.
```

Reply:
Let's also fix that indent above on line # 65.

Reply:
Oh, yeah, we should also remove the italic underscores as per Products Team

Reply:
whoops please disregard my last comment for this case